css/secondary-page.styl

```diff
@@ -53,6 +53,7 @@
       background-color: #E8E8E8
       border-radius: 4px
       padding: 1em
+      overflow: auto
```

Comment:
Tiny quibble, but our convention is to put css properties in alphabetic order (you can find this info in the repo readme).